mmv1/products/workstations/WorkstationConfig.yaml

```diff
@@ -314,6 +337,7 @@ properties:
           If set, overrides the USER specified in the image with the given uid.
   - !ruby/object:Api::Type::NestedObject
     name: 'encryptionKey'
+    immutable: true
```

Comment:
Can you verify that every possible update to this field would have resulted in an error? That means going to/from all of:

* Having the field undefined in config
* Having an empty string in config
* Having a valid value in config

(going to an invalid value would presumably have always failed, so we can omit that)